fedrec/user_modules/envis_trainer.py

```diff
@@ -41,6 +89,50 @@ def check_only_one_declaration(instance, _, value):
 
 
 class EnvisTrainer(EnvisBase):
+    """
+    This class is used to train the model. It is responsible for training the
```

Comment:
```suggestion
    EnvisTrainer is used to train the model. It is responsible for training the
```
Please don't use `responsible` every time. You can come up with another alternative like `in charge of` and some phrases like that are more technical.